bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -116,7 +118,6 @@ public ShellyBaseHandler(Thing thing, ShellyBindingConfiguration bindingConfig,
      */
     @Override
     public void initialize() {
-        updateStatus(ThingStatus.UNKNOWN);
```

Comment:
Why did you remove this line?

Reply:
I keep the status INITIALIZING until the device is fully initialized. For non-battery devices this will be done with the next status update, for battery-powered devices this could take longer until the device wakes up. UNKOWN sounds like an error, even I now that the device is not yet initialized

Reply:
I am pretty sure that we discussed this before (when adding that line), but I can certainly again point you to the developer docs: https://github.com/openhab/openhab-core/blob/master/tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__Handler.java#L69-L79

Reply:
ok, will be changed

Reply:
I don't see any change?

Reply:
done